src/pages/Payment/PaymentComponents/PaymentPage/PaymentScreen/Paymentscreenright.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import styled from 'styled-components';
+import { loadTossPayments } from '@tosspayments/payment-sdk';
+
+const REACT_APP_TOSS = `${process.env.REACT_APP_TOSS}`;
+
+const onclickHandler = paymentType =>
+  loadTossPayments(REACT_APP_TOSS).then(tossPayments => {
+    tossPayments.requestPayment('카드', {
+      // 결제 수단
+      // 결제 정보
+      amount: 1,
+      orderId: 'LgAY7eY5rKzwB-x901h2t',
+      orderName: '토스 티셔츠 외 2건',
+      customerName: '박토스',
+      successUrl: 'http://localhost:3000/paymentsuccess',
+      failUrl: 'http://localhost:3000/payment',
+      flowMode: 'DIRECT',
+      easyPay: paymentType,
+    });
+  });
+
+const Paymentscreenright = () => {
+  return (
+    <Rightsidecontainer>
+      <Pledgewrapper>
+        <Pledgeamountarea>
+          <Pledgetotalwrap>
+            <Pledgeamounttitle>최종 후원 금액</Pledgeamounttitle>
+            <Pledgeamountdesc>
+              <span>59,600</span>
+              {' 원'}
+            </Pledgeamountdesc>
+          </Pledgetotalwrap>
+        </Pledgeamountarea>
+      </Pledgewrapper>
+      <Paybuttons>
+        <Solidbutton
+          onClick={() => {
+            onclickHandler('토스페이');
+          }}
+        >
+          <Tosspayimg
+            alt="토스 페이"
+            src="/images/Payment/Toss_Symbol_Primary.png"
+          />
+          <span>토스페이로 후원하기</span>
+        </Solidbutton>
```

Comment:
```suggestion
 {PAYMENT_LIST.map(({ id, title, img, width }) => {
          return (
            <Solidbutton
              key={id}
              onClick={() => {
                onclickHandler(title);
              }}
            >
              <Payimg
                alt={title}
                src={`/images/Payment/${img}.png`}
                width={width}
              />
              <span>{title}로 후원하기</span>
            </Solidbutton>
          );
        })}
```

중복되는 UI 부분을 위와 같이 상수데이터를 활용해 줄일 수 있겠습니다.

상수데이터의 예시는 아래를 참고해주세요

```
const PAYMENT_LIST = [
  { id: 1, title: '토스페이', img: 'Toss_Symbol_Primary', width: '39px' },
  { id: 2, title: '네이버페이', img: 'npay_78', width: '55px' },
  {
    id: 3,
    title: '카카오페이',
    img: 'payment_icon_yellow_small',
    width: '55px',
  },
  { id: 4, title: '삼성페이', img: 'samsungpay', width: '55px' },
];
```

Reply:
넵 수정하였습니다.